File: hello.java
import java.util.Scanner;

public class AddTwoNumbers {
    public static void main(String[] args) {
        // Create a Scanner object to take input
        Scanner scanner = new Scanner(System.in);

        // System.out.print("Enter first number: ");
        int num1 = scanner.nextInt();  // Read first number

        // System.out.print("Enter second number: ");
        int num2 = scanner.nextInt();  // Read second number

        int sum = num1 + num2;  // Add the numbers

        System.out.println("Sum: " + sum);  // Print the result

        scanner.close();  // Close the scanner
    }
}

File: hello.rs
use std::io;

fn main() {
    let mut input1 = String::new();
    let mut input2 = String::new();

    io::stdin().read_line(&mut input1).expect("Failed to read input");

    io::stdin().read_line(&mut input2).expect("Failed to read input");

    let num1: i32 = input1.trim().parse().expect("Please enter a valid number");
    let num2: i32 = input2.trim().parse().expect("Please enter a valid number");

    let sum = num1 + num2;
    println!("Sum: {}", sum);
}
